src/rules.jl

```diff
@@ -0,0 +1,64 @@
+## Square
+SQUARE_PAIRS = (
+    (sqrt2, 2.0),
+    (sqrt3, 3.0),
```

Comment:
I'm not sure these should be `(sqrt2, 2.0), (sqrt3, 3.0)` or `(sqrt2, 2), (sqrt3, 3)`. I was referring to `sin(π)`.
```julia
julia> versioninfo()
Julia Version 1.9.0-DEV.428
Commit f6a95348fe (2022-04-21 11:53 UTC)
Platform Info:
  OS: Linux (x86_64-pc-linux-gnu)
  CPU: 16 × AMD Ryzen 7 2700X Eight-Core Processor
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-13.0.1 (ORCJIT, znver1)
  Threads: 1 on 16 virtual cores

julia> sin(π)
0.0

julia> cos(π)
-1.0
```

Reply:
I think they should be `Float64`, as explained in my other comment.